lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -503,8 +503,11 @@ export class ResponseHandler {
         let familyId: string = Constants.EMPTY_STRING;
 
         if (cacheRecord.accessToken) {
+            // if the request object has `reqCnf` property, the token will be returned unsigned
             if (
-                cacheRecord.accessToken.tokenType === AuthenticationScheme.POP
+                cacheRecord.accessToken.tokenType ===
+                    AuthenticationScheme.POP &&
+                !request.reqCnf
```

Comment:
Is the request here the original request or the "validatedRequest"? If it's the latter this will always be defined, right?

Reply:
Good q. Let me verify that.

Reply:
`request` and `request.code` are the only data checked  prior to this. Since `reqCnf` is never defined unless explicitly set by the user, the check should still hold.

Reply:
`request` is ensured to be defined by the time we reach here. However `request.reqCnf` should still be undefined unless set explicitly. Does that clarify?

Reply:
I'm not sure I believe you :) Can we add a test for the msal-browser APIs to verify? The first thing we do in the top level APIs is create a validated request object which should create a reqCnf if not already provided. The original request object is never passed down into msal-common AFAIK

Reply:
Ha ha, Sure, lets add tests.

Reply:
Added tests. Please check.